domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -147,6 +153,16 @@ class CameraXCameraUseCase @Inject constructor(
         recording?.stop()
     }
 
+     override fun setFlashMode(flashModeStatus: FlashModeStatus){
+        val newFlashMode = when (flashModeStatus) {
+            FlashModeStatus.OFF -> ImageCapture.FLASH_MODE_OFF // 2
+            FlashModeStatus.ON -> ImageCapture.FLASH_MODE_ON // 1
+            FlashModeStatus.AUTO -> ImageCapture.FLASH_MODE_AUTO // 0
+        }
+        imageCaptureUseCase.flashMode = newFlashMode
+        Log.d(TAG, "Set flash mode to: " + imageCaptureUseCase.flashMode)
```

Comment:
nit: prefer using string templates rather than string concatenation or string format here
https://kotlinlang.org/docs/strings.html#string-templates

```
Log.d(TAG, "Set flash mode to: ${imageCaptureUseCase.flashMode}")
```